did-pkh-method-draft.md

```diff
@@ -0,0 +1,256 @@
+Verifiable Credentials or Authorization Capabilities, and produce proper
+signature-suite definitions, such as "metamask-signing" (signing according to
+the [[eip712]] protocol, soon to be a work item at W3C-CCG).
+
+### Relationship to other DID architectures
+
+did:pkh bears many similarities to
+[did:key](https://w3c-ccg.github.io/did-method-key/#introduction) except it is
+optimized for identifiers derived from hashes of public keys according to
+well-known algorithms (commonly referred to as "public key hashes" because in
+most cases they are a public key hashed according to a standard hash function).
+
+### Combination with other DID methods
+
+Another difference from did:key is that did:pkh is design to have many "upgrade
+paths" for DIDs deterministically generated from existing keypairs.  Namely:
+- if a did:pkh is controlled by a keypair which is valid for generating a
+  blockchain-published DID document according to another method (for instance,
+  did:tz, did:btcr or did:ethr), its did document can be translated to the form
+  of that method's documents, and it can be registered there.
+
+## Design Goals
+
+1. The primary goal of this method is to allow any valid blockchain address to
+   "spin up" a feature-limited but valid and widely interoperable DID and DID
+   Document, valid in a limited context where accounts are represented by DIDs. 
+2. This method is very narrow and unopinionated to allow a wide range of
+   implementations.
+3. For example, the validity of each address to be wrapped in a DID is checked
+   according to the [CAIP-10] standard before generating, to prevent a did:pkh
+   being presented as valid that would not be on its corresponding blockchain.
+   **No further validation** is assumed or provided in the reference
+   implemention, but implementers may still choose to gate generation to
+   on-chain accounts or balance-holding accounts as per the requirements of
+   their specific use case.
+4. As this method is designed for interoperability with blockchain web wallets,
+   authentication and signing functions are left to the blockchain-specific
+   capabilities of the wallets supported by a given implementation, "dApp", or
+   context. This has implications for the degree of privacy and security that
+   can be assumed. Importantly, these vary across blockchains so some use-cases
+   may choose to treat PKHs differently per prefix.
+
+## Identifier scheme
+
+### Syntax and Interpretation
+
+```
+pkh-did    = "did:pkh:" address
+address    = account_address according to [CAIP-10]
+```
+
+### Examples
+
+Here is an example from each currently supported network, linked to a sample
+JSON-LD DID document derived from each:
+
+| Network (key type) | example (linked to sample DID document) |
+|:---:|:---:|
+| btc | [did:pkh:bip122:000000000019d6689c085ae165831e93:128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6](https://github.com/spruceid/ssi/blob/main/did-pkh/tests/did-btc.jsonld) |
```

Comment:
I suppose these example links should go to this repo?

Reply:
i agree, but it might be easier (and more politick) to merge as is and open a separate issue.  feel free to assign that issue to me!

Reply:
https://github.com/w3c-ccg/did-pkh/pull/7